securedrop/tests/test_journalist.py

```diff
@@ -2728,56 +2894,6 @@ def test_valid_user_invalid_first_last_name_change(config, journalist_app, test_
                 )
 
 
-def test_regenerate_totp(journalist_app, test_journo):
-    old_secret = test_journo["otp_secret"]
-
-    with journalist_app.test_client() as app:
-        login_journalist(
-            app,
-            test_journo["username"],
-            test_journo["password"],
-            test_journo["otp_secret"],
-        )
-
-        with InstrumentedApp(journalist_app) as ins:
-            resp = app.post(url_for("account.reset_two_factor_totp"))
-
-            new_secret = Journalist.query.get(test_journo["id"]).otp_secret
-
-            # check that totp is different
-            assert new_secret != old_secret
-
-            # should redirect to verification page
-            ins.assert_redirects(resp, url_for("account.new_two_factor"))
-
-
-def test_edit_hotp(journalist_app, test_journo):
-    old_secret = test_journo["otp_secret"]
-    valid_secret = "DEADBEEF01234567DEADBEEF01234567DADEFEEB"
-
-    with journalist_app.test_client() as app:
-        login_journalist(
-            app,
-            test_journo["username"],
-            test_journo["password"],
-            test_journo["otp_secret"],
-        )
-
-        with InstrumentedApp(journalist_app) as ins:
-            resp = app.post(
-                url_for("account.reset_two_factor_hotp"),
-                data=dict(otp_secret=valid_secret),
-            )
-
-            new_secret = Journalist.query.get(test_journo["id"]).otp_secret
-
-            # check that totp is different
-            assert new_secret != old_secret
-
-            # should redirect to verification page
-            ins.assert_redirects(resp, url_for("account.new_two_factor"))
```

Comment:
Removed these tests since they seem to be duplicates of `test_user_resets_totp` and `test_user_resets_hotp`